pipeline_steps/common.groovy

```diff
@@ -22,10 +22,10 @@ def install_ansible(){
     source .venv/bin/activate
 
     # These pip commands cannot be combined into one.
-    pip install -U six packaging appdirs
-    pip install -U setuptools
-    pip install 'pip==9.0.1'
+    pip install --index-url https://pypi.python.org/simple 'pip==9.0.1 setuptools==33.1.1'
```

Comment:
this fails when trying it on trusty with the python-pip package installed

so it looks like this will have to do:

-    pip install -U six packaging appdirs		 +    pip install --index-url https://pypi.python.org/simple 'pip==9.0.1 setuptools==33.1.1'
  
            
  Write  Preview

this fails when trying it on trusty with the python-pip package installed

so it looks like this will have to do:

pip install -U --index-url https://pypi.python.org/simple six packaging appdirs
pip install -U --index-url https://pypi.python.org/simple 'setuptools==33.1.1'
pip install --index-url https://pypi.python.org/simple 'pip==9.0.1'
...